dataplane/api-server/src/lib.rs

```diff
@@ -24,9 +24,15 @@ pub async fn start(
     gateway_indexes_map: HashMap<MapData, BackendKey, u16>,
     tcp_conns_map: HashMap<MapData, ClientKey, LoadBalancerMapping>,
 ) -> Result<(), Error> {
+    let (mut health_reporter, health_service) = tonic_health::server::health_reporter();
+    health_reporter
+        .set_serving::<BackendsServer<server::BackendService>>()
+        .await;
```

Comment:
don't think this is required, since we are not specifying the service name in any of the probes anyway.

Reply:
Modified in https://github.com/kubernetes-sigs/blixt/pull/182/commits/82c634e3ca82b376d9266ce8c68d48e01c530e3c